core/test/utils/matrix_generator_test.cpp

```diff
@@ -91,8 +91,8 @@ class MatrixGenerator : public ::testing::Test {
 
 TEST_F(MatrixGenerator, OutputHasCorrectSize)
 {
-    EXPECT_EQ(mtx->get_num_rows(), 500);
-    ASSERT_EQ(mtx->get_num_cols(), 100);
+    EXPECT_EQ(mtx->get_size().num_rows, 500);
```

Comment:
Should this be replaced with a tuple-comparison?

Reply:
It _could_ be, to simplify the code a bit. Just to document it for the future, had to come up with something like this to do it everywhere:

```bash
find . -type f -not -path "*.git/*" -exec perl -i -0pe 's/(EXPECT|ASSERT)_EQ\((\w+)->get_size\(\)\.num_rows, (\d+)\);\n\s*(EXPECT|ASSERT)_EQ\(\w+->get_size\(\)\.num_cols, (\d+)\)/ASSERT_EQ(\2->get_size(), gko::dim(\3, \5))/g' {} + 
```

Reply:
it's fine!